tests/unit/SbcFeeSummary.spec.ts

```diff
@@ -1,378 +0,0 @@
-import Vue from 'vue'
```

Comment:
Removed as it should be in sbc-common-components library instead.

https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13762